src/mpyl/steps/deploy/cloudfront_kubernetes_deploy.py

```diff
@@ -0,0 +1,54 @@
+""" Step that uploads static files to s3 and then deploys the docker image produced in the build stage to Kubernetes,
+using HELM."""
+from logging import Logger
+import tempfile
+
+from .kubernetes import DeployKubernetes
+from .. import Step, Meta
+from ...project import Stage
+from ..models import Input, Output, ArtifactType
+from ...utilities.docker import docker_image_tag, docker_copy
+from ...utilities.s3 import S3Client, S3ClientConfig
+
+STATIC_FOLDER = 'static'
+
+
+class CloudFrontKubernetesDeploy(Step):
```

Comment:
Why still the name CloudFrontKubernetesDeploy? Why nog something like DeployS3?

Reply:
I originally thought that would be a more appropriate name but since the step also calls the Kubernetes Deploy step, and the s3 key structure is specific to our cloudfront setup, it seems to be the best name